functions/src/modules/pm/categories/getCategory.ts

```diff
@@ -1,5 +1,46 @@
-import { ExpressFunction } from "../../../@types";
+import type { ExpressFunction } from "../../../@types";
+import { firebaseApp } from "../../../config/firebaseConfig";
+import { expressErrorHandlerFactory } from "../../../helpers";
+import { uasPermissionSwitch } from "../../../systems/uas";
 
-const replaceMe: ExpressFunction = (req, res, next) => {};
+const getCategory: ExpressFunction = (req, res, next) => {
+  uasPermissionSwitch({
+    organizer: { accepted: execute },
+    sponsor: { accepted: executeIfCategoryMatches },
+  })(req, res, next);
+};
 
-export default replaceMe;
+const execute: ExpressFunction = (req, res, next) => {
+  const errorHandler = expressErrorHandlerFactory(req, res, next);
+  firebaseApp
+    .firestore()
+    .collection("prizeCategories")
+    .doc(req.params.prizeCategoryId)
```

Comment:
Once again, we should check what the final parameter is, but I think it is just `categoryId`.
```suggestion
    .doc(req.params.categoryId)
```